base/applications/rapps/include/rosui.h

```diff
@@ -97,17 +104,14 @@ class CPointerArray
     BOOL RemoveAt(INT i)
     {
         PVOID ptr = DPA_DeletePtr(m_hDpa, i);
-        if (ptr != NULL)
-        {
-            OnRemoveItem(reinterpret_cast<T*>(ptr));
-            return TRUE;
-        }
-        return FALSE;
+        if (!ptr)
+            return FALSE;
+        return OnRemoveItem(reinterpret_cast<T*>(ptr));
     }
 
     BOOL Clear()
     {
-        DPA_EnumCallback(s_OnRemoveItem, this);
+        DPA_EnumCallback(m_hDpa, s_OnRemoveItem, this);
```

Comment:
Missing parameter, this was an unseen compilation error because `Clear()` was never used in RAPPS.